WordPress/Classes/ViewRelated/Comments/CommentTableHeaderView.swift

```diff
@@ -96,25 +96,14 @@ private struct CommentHeaderView: View {
     @State var showsDisclosureIndicator = true
 
     var body: some View {
-        if #available(iOS 15.0, *) {
-            // Material ShapeStyles are only available from iOS 15.0.
-            content.background(.ultraThinMaterial)
-        } else {
-            ZStack {
-                VisualEffectView(effect: UIBlurEffect(style: .systemUltraThinMaterial))
-                content
-            }
-        }
-    }
-
-    var content: some View {
         HStack {
             text
             Spacer()
             if showsDisclosureIndicator {
                 disclosureIndicator
             }
         }
+        .background(.ultraThinMaterial)
         .padding(EdgeInsets(top: 10, leading: 16, bottom: 10, trailing: 16))
```

Comment:
The `background` location appears to matter here:

| Location | Screenshot |
|------|-------------|
| `background` above `padding` (current) | ![above](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/7a6a5ae2-363a-41ea-b4b7-3e60e5a4020b) |
| `background` below `padding` | ![below](https://github.com/wordpress-mobile/WordPress-iOS/assets/2454408/6c3c3ce1-55a1-4d5b-a9b0-9e25fdaafeb8) |

The original was placed below the `padding` since it was `content.background(.ultraThinMaterial)`. So I think the code should be:

```suggestion
        .padding(EdgeInsets(top: 10, leading: 16, bottom: 10, trailing: 16))
        .background(.ultraThinMaterial)
```

Reply:
Good catch! This is fixed in https://github.com/wordpress-mobile/WordPress-iOS/pull/20981/commits/48eea91b88050bd5a7bee90bd831b4fd073933ea.